.readthedocs.yml

```
@@ -0,0 +1,41 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html
+# for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  builder: dirhtml
+  configuration: docs/conf.py
+  fail_on_warning: true
+
+# Build documentation with MkDocs
+#mkdocs:
+#  configuration: mkdocs.yml
+#  fail_on_warning: true
+
+# Optionally build your docs in additional formats
+# such as PDF and ePub
+formats: []
+
+submodules:
+  include: all # []
+  exclude: []
+  recursive: true
+
+build:
+  image: latest
+
+# Optionally set the version of Python and requirements required
+# to build docs
+python:
+  version: 3.7
```

Let's use something newer now:
```suggestion
python:
  version: 3.8
```